### PR TITLE
Add: PassengersInputs component

### LINKED ISSUE #32: Create the PassengersPicker Component
## PassengersPicker

- [x] After the user clicks on the Passengers Input field, it will open the PassengersPicker component.
- [x] Use the `universal-components` library for Stepper and Text.
- [x] Put the negative value is not allowed.

### before click:
screen shot 2018-12-04 at 14 03 35

### after click:
passengerspicker

### PR DESCRIPTION
**Pull request by JoseDuSV**
To prevent confusion, name was changed (after short investigation) to `PassengersInputs` because there is already different component called `PassengerPicker` in designs - https://www.figma.com/file/WS4aFTwDfDHv6FL5Qawb0PR8/Passenger-Picker?node-id=0%3A1 .
But I'm definitely open to any other suggestions.

Closes #32

### CODE REVIEW

**Comment by FilipMessa** [comment]
👍

File: packages/components/src/passengersInputs/__tests__/PassengersInputs.test.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+const { getByText } = render(
+  <PassengersInputs
+    {...testData0}
+    onClosePress={onClose}
+    onSavePress={onSave}
+  />,
+);
+
+it('should execute onClosePress method', () => {
+  fireEvent(getByText('Close'), 'press');
```
[the remark this answers is not in the document]
Comment (JoseDuSV) [reply packages/components/src/passengersInputs/__tests__/PassengersInputs.test.js]: Updated

File: packages/components/src/passengersInputs/__tests__/PassengersInputs.test.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        onClosePress={jest.fn()}
+        onSavePress={jest.fn()}
+      />,
+    ),
+  ).toMatchSnapshot();
+});
+
+test('updateAdults > should return correct value on increment', () => {
+  expect(updateAdults(testData0, 1)).toMatchObject(testData0);
+});
```
[the remark this answers is not in the document]
Comment (JoseDuSV) [reply packages/components/src/passengersInputs/__tests__/PassengersInputs.test.js]: Sure. There are multiple edge cases in this function so I extended tests little bit to cover scenarios when limits are reached and when not.

[the remark this answers is not in the document]
Comment (FilipMessa) [reply packages/components/src/passengersInputs/__tests__/PassengersInputs.test.js]: 👍
